build.gradle

```diff
@@ -1170,7 +1171,46 @@ project(':streams') {
     duplicatesStrategy 'exclude'
   }
 
+  task determineCommitId {
```

Comment:
The determineCommitId logic is the same for both client and streams, I'm wondering if we can make it a global task that both can rely on (I'm not an expert of gradle, just wondering).

Reply:
I also had the same thought and I am also not a gradle expert. I would leave the duplicate code as it is for now to have a chance to get client-level metrics into the release and fix this afterwards.